### DISCUSSION TITLE
how to set sys.MEIPASS path

### DISCUSSION THREAD

**Discussion by Boris-code** [pyinstaller/pyinstaller#7449] [PyInstaller] [answered]
When sys.MEIPASS  contains chinese path, then load dll may be failed.  Can I specify the path？

**Comment by rokm** [comment, marked as answer]
No, `sys._MEIPASS` is set automatically. 

In `onedir` builds, it corresponds to the directory where executable is located (i.e., the application directory). 

In `onefile` builds, it corresponds to the temporary directory where application unpacks itself. You can override this with [`--runtime-tmpdir`](https://pyinstaller.org/en/stable/usage.html#cmdoption-runtime-tmpdir) option at build time; the default uses `TMP` environment variable, so you can also try changing that prior to launching the application.

**Comment by Boris-code** [comment]
thanks
